plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/custom/configuration/operator/transformers/AbstractDMOperatorTransformer.java

```diff
@@ -69,5 +69,15 @@ protected String appendOutputVariable(DMOperation operation, List<DMVariable> ou
 		}
 		return operationBuilder.toString();
 	}
+	
+	protected SchemaDataType getOutputVariableType(List<DMVariable> outputVariables)
+			throws DataMapperException {
+		int numOfOutputVariables = outputVariables.size();
+		SchemaDataType outputDataType = null;
+		for (int variableIndex = 0; variableIndex < numOfOutputVariables; variableIndex++) {
+			outputDataType = outputVariables.get(variableIndex).getSchemaVariableType();
```

Comment:
outputDataType will always be replaced with (numOfOutputVariables-1) th element. Do we need the for loop?

Reply:
Corrected.